packages/web-react/src/components/Navigation/demo/NavigationButtons.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Button } from '../../Button';
+import Navigation from '../Navigation';
+import NavigationItem from '../NavigationItem';
+
+const NavigationDefault = () => {
+  return (
+    <div style={{ '--spirit-header-height': '72px' } as React.CSSProperties}>
```

Comment:
**question (non-blocking):** Can you please explain this to me?

Reply:
NavigationLink can inherit height so it is as big as its parent. Here it is not in Header, so it is not in inheriting it from there, so we set it manually.

Reply:
Could we please make a React component from it? We are repeating the code. And maybe add your explanation to it. I know it is for demo purposes only. We have a few such components.

Reply:
> can inherit height

I noted in the `web` PR #1795 there could be a default height in the `Navigation` styles. If we agree on it, this element could go away.

Reply:
removed